crypten/mpc/primitives/resharing.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import crypten.communicator as comm
+import torch
+
+
+def replicate_shares(x_share):
+    rank = comm.get().get_rank()
+    world_size = comm.get().get_world_size()
+    prev_rank = (rank - 1) % world_size
+    next_rank = (rank + 1) % world_size
+
+    x_rep = torch.zeros_like(x_share)
+
+    req0 = comm.get().isend(x_share, dst=next_rank)
+    req1 = comm.get().irecv(x_rep, src=prev_rank)
+
+    req0.wait()
+    req1.wait()
+
+    return x_rep
+
+
+def __replicated_secret_sharing_protocol(op, x, y, *args, **kwargs):
+    assert op in {
```

Comment:
NIT: Assert is a good idea, but if ```CrypTen``` becomes production like ready maybe an exception ```TypeError``` would be better. If you run python with optimization flags, this will get thrown away.
Reference: http://archive.is/5GfiG

Reply:
Hi Gmuraru, thanks for your review. We've already abandoned this implementation of RSS and plan to come up with a new implementation later this year.